include/stratosphere/mitm/mitm_server.hpp

```diff
@@ -97,16 +103,16 @@ class MitmServer : public IWaitable {
             
             smMitMExit();
                         
-            this->GetSessionManager()->AddWaitable(new MitmSession(session_h, client_pid, forward_service, std::make_shared<T>(forward_service, client_pid)));
+            this->GetSessionManager()->AddWaitable(new MitmSession(session_h, client_pid, forward_service, ConstructionDetailHelper(forward_service, client_pid, std::forward<std::tuple<Args...>>(args), std::index_sequence_for<Args...>())));
```

Comment:
`std::forward` is misleading here - you're not applying it to a reference parameter, so you can replace it with a simple `std::move` (if that's what you intended, of course).